calculator/math_parser.py

```diff
@@ -122,3 +127,22 @@ def evaluate(expression):
         apply_operation(op_stack, out_stack)
 
     return out_stack[-1]
+
+
+def main():
+    """
+        simple user-interface
+    """
+    print ("\t\tCalculator\n\n")
+    userInput = input("expression or exit: ")
+    while userInput != "exit":
+        try:
+            print ("The result is {0}".format(evaluate(userInput)))
+        except Exception:
+            print ("invalid syntax!")
+        userInput = input("expression or exit: ")
+    print ("program end")

```

Comment:
please remove the space between `print` and `(` at all usages.

Reply:
OK,